scripts_generate/build.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import multiprocessing
+
+from pull_assets import pull_all
+from create_detection_data import generate_all_images as create_det_data
+from create_clf_data import create_clf_images as create_clf_data
+
+
+if __name__ == "__main__":
+    # Try to get env vars from tox build
+    try:
+        NUM_IMAGES = int(os.environ["NUM_IMAGES"])
+        NUM_VAL_IMAGES = int(os.environ["NUM_VAL_IMAGES"])
+    except Exception:
+        NUM_IMAGES = 20
+        NUM_VAL_IMAGES = 10
```

Comment:
I think one of these should be from the config yaml.

Reply:
Good catch